simulation/utils/helpers.py

```diff
@@ -95,3 +95,22 @@ def from_iso_to_unix_time(iso_time):
     unix_time = int(dt.timestamp())
 
     return unix_time
+
+
+def timeout_from_start_time(start_time_str: str) -> float:
+    """
+    Calculate the timeout duration from the start_time to the current time.
+
+    :param start_time: ISO 8601 string representing the start time.
+    :return: Timeout duration in seconds.
+    """
+    # Convert start_time to a datetime object
+    start_time = datetime.fromisoformat(start_time_str)
+
+    # Get current date and time
+    current_time = datetime.now(timezone.utc)
+
+    # Calculate the timeout duration
+    timeout_duration = (start_time - current_time).total_seconds()
```

Comment:
can we add the configuration param I introduced before:
```
--neuron.timeout
```

so the
```
timeout_duration = max((start_time - current_time).total_seconds(), base_neuron.config.neuron.timeout)
```

or
```
dynamic_timeout_duration = (start_time - current_time).total_seconds()
timeout_duration = max(dynamic_timeout_duration, base_neuron.config.neuron.timeout)
```

or we can make some logic that substitute the dynamic timeout by configuration timeout or takes minimum - not sure really what is the best alternative, but we can start with something like you did anyway and add the `received_at` column into the `validator_requests` table, when miners actually responded

Reply:
Ha yes, I missed about base_neuron.config.neuron.timeout